include/net/fota_download.rst

```diff
@@ -1,46 +1,59 @@
 .. _lib_fota_download:
 
-FOTA download
-#############
+FOTA download library
+#####################
 
 .. contents::
    :local:
    :depth: 2
 
 The firmware over-the-air (FOTA) download library provides functions for downloading a firmware file as an upgrade candidate to the DFU target that is used in the :ref:`lib_dfu_target` library.
 
-To start a FOTA download, you must provide the URL for the file that should be downloaded.
-The required arguments are ``host`` (for example, ``my_domain.com``) and ``file`` (for example, ``path/to/resource/file.bin``).
+Configuration and implementation
+********************************
 
-.. note::
-   To upgrade the upgradable bootloader, you must provide two firmware files: one linked against the S0 partition and one linked against the S1 partition.
-   See :ref:`upgradable_bootloader` for more information.
+To start a FOTA download, provide the URL for the file that should be downloaded, specifying the required arguments as follows:
 
-   To do this, you must provide two paths in the ``file`` argument, separated by a space character.
-   The first entry  must point to the MCUboot variant linked against the S0 partition.
-   The second entry  must point to the MCUboot variant linked against the S1 partition.
+   * ``host`` - It indicates the hostname or the IPv4 address of the target host.
+     For example, ``my_domain.com``.
+   * ``file`` - It indicates the path to the file.
+     For example, ``path/to/resource/file.bin``.
```

Comment:
@hakonfam Is this the path within the host location?

Reply:
Yes, so `my_domain.com/path/to/resource/file.bin`